src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -25,92 +25,135 @@ import upickle.default.write
 import scala.reflect.ClassTag
 import scalaz.concurrent.Task
 
-case class GetManyDocumentsQueryBuilder(
+sealed trait DocsInResult
+abstract class IncludeDoc[D: R] extends DocsInResult
+abstract class ExcludeDocs extends DocsInResult
+
+sealed trait MissingIdsInQuery
+trait MissingAllowed extends MissingIdsInQuery
+trait MissingNotAllowed extends MissingIdsInQuery
+
+sealed trait DocType
+abstract class ForDocType[D: R, K: R, V: R] extends DocType
+abstract class AnyDocType extends DocType
+
+case class GetManyDocumentsQueryBuilder[ID <: DocsInResult, AM <: MissingIdsInQuery, BT <: DocType](
```

Comment:
were you referring to the constructor? if so yes... but probably not the case class itself since its type is required outside this class